ui/components/app/transaction-list-item-details/index.scss

```diff
@@ -64,10 +62,9 @@
   }
 
   &__operations {
-    margin: 0 16px 16px 16px;
+    margin: 0 0 16px 16px;
```

Comment:
![CleanShot 2025-04-11 at 09 40 46](https://github.com/user-attachments/assets/c5292ceb-9911-46ec-bf1a-e5465bbd9935)
![CleanShot 2025-04-11 at 09 40 59](https://github.com/user-attachments/assets/54212bff-e75c-468c-9de9-ce5be47ebe6d)

we can resurrect this to fix an outstanding, no padding issue